system_tests/common_test.go

```diff
@@ -1718,7 +1718,7 @@ func logParser[T any](t *testing.T, source string, name string) func(*types.Log)
 // recordBlock writes a json file with all of the data needed to validate a block.
 //
 // This can be used as an input to the arbitrator prover to validate a block.
-func recordBlock(t *testing.T, block uint64, builder *NodeBuilder, blockInputJSONPath string) {
+func recordBlock(t *testing.T, block uint64, builder *NodeBuilder, targets []ethdb.WasmTarget, blockInputJSONPath string) {
```

Comment:
It might be nice to put the targets argument at the end and change it's type to ...ethdb.WasmTarget so that callers don't have to create a slice if they don't want to.

Reply:
thats a great point, fixed it!